src/main/java/frc/robot/subsystems/Magazine.java

```diff
@@ -13,7 +13,7 @@
  */
 
 public class Magazine extends PIDSubsystem {
-    private WPI_TalonFX magazineMotor = new WPI_TalonFX(Constants.Motors.magazineMotorID);
+    public WPI_TalonFX magazineMotor = new WPI_TalonFX(Constants.Motors.magazineMotorID);
```

Comment:
Don't make it public.  Just create a couple of methods in teh subsystem to turn on and off the motor (non-pid)